### PR TITLE
feat: Add integration tests for bridges

### PR DESCRIPTION
**Pull request by jrconlin**
Integration tests patch as close to send as possible.

issue: #692

### CODE REVIEW

**Comment by bbangert** [comment]
Waffle doesn't seem to be linking this, probably because the commit format has a colon in it when it shouldn't, per contributing:
```
In order to maintain a reference to the context of the commit, add closes
 #<issue_number> if it closes a related issue or issue #<issue_number> if 
it's a partial fix.
```

**Comment by jrconlin** [comment]
crap, must have gotten in there from vim.

File: autopush/router/apns2.py [hunk 1]
```diff
@@ -57,24 +58,27 @@ def __init__(self, cert_file, key_file, topic,
         :type logger: logger
         :param metrics: Metric recorder
         :type metrics: autopush.metrics.IMetric
+        :param load_connections: used for testing
+        :type load_connections: bool
```
Comment (bbangert) [inline autopush/router/apns2.py]: nit: blank line before closing triple-quote

Comment (jrconlin) [inline autopush/router/apns2.py]: ✅

**Pull request merged**